rootfs/etc/systemd/system/dietpi-postboot.service

```diff
@@ -2,7 +2,7 @@
 Description=DietPi-PostBoot
 # Order 4
 Requires=dietpi-boot.service
-After=dietpi-boot.service multi-user.target
+After=dietpi-boot.service
```

Comment:
@Fourdee 
This was for the following reason:
- If `dietpi-postboot.service` runs before `multi-user.target`, the Debian login prompts will override our own login prompt and info. With `After=multi-user.target` is it assured that Debian login prompts come first (clearing the console and such), then out own login prompt will be assured to show and output is visible on console, including service starts and such.
- However indeed in case of autologin (most `dietpi-autostart` options, but 0) this is indeed suboptimal, since login is done directly after multi-user.target (respectively already with getty@tty1.service), so DietPi-Login for sure will run into the loop waiting for PostBoot to finish.
- I opened this issue to think about a proper solution: https://github.com/Fourdee/DietPi/issues/2476
E.g. Use `After=multi-user.target` only in case of manual console login and remove it if any autologin is active. Also skip our login prompt then, since it is obsolete.